src/scripts/index.js

```diff
@@ -102,3 +102,31 @@ function createProduct(evt) {
   toggleProductModal();
   productForm.reset();
 }
+
+function showAlert({ type, productName }) {
```

Comment:
Como a gente não está usando uma tipagem ou algo do tipo, as vezes vale a pena implementar uma validação que se o type não for de tal, tal tipo, lança um Error.

Eu alteraria o nome dessa função para showProductsAlert ou tentaria deixar ela ainda mais genérica, passando a message e o title como parâmetros também. E de repente isolando os valores em constantes.